bin/test

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+case `uname -s` in
+  Linux )
+    XDG_DATA_HOME="${XDG_DATA_HOME:-${HOME}/.local/share}"
+    KSPDIR="${KSPDIR:-${XDG_DATA_HOME}/Steam/SteamApps/common/Kerbal Space Program}"
+    MANAGED="${KSPDIR}/KSP_Data/Managed"
+    ;;
+  Darwin )
+    KSPDIR="${KSPDIR:-/Users/lamont/Library/Application Support/Steam/steamapps/common/Kerbal Space Program}"
+    MANAGED="${KSPDIR}/KSP.app/Contents/Resources/Data/Managed"
+    ;;
+  * )
+    echo "this may go badly for you..."
+    ;;
+esac
+
+nuget restore MechJeb2.sln
+msbuild /p:Configuration=Release /p:ReferencePath="${MANAGED}" MechJeb2.sln
+
+runtest="mono ./packages/xunit.runner.console.*/tools/*/xunit.console.exe"
+
+echo "Running tests..."
+
+$runtest ./MechJeb2Tests/bin/Release/MechJeb2Tests.dll
```

Comment:
Interesting. When I added the test in Jenkins for MM it did not want to work until I copied all of the xunit console stuff in the working dir. I wonder why it works here and not on my jenkins...

Reply:
It may need copy local set to true for all the relevant DLLs.

Reply:
the msbuild goes through and copies all the xunit dlls to the bin/Release directory, so it gets picked up out of the working dir here.  i think there's some stuff in the MechJeb2Tests.csproj that wires this up

Reply:
it actually copies all the unity DLLs as well, and since I'm just pointed at my KSP dir it looks like this:

```
% ls MechJeb2Tests/bin/Release
Assembly-CSharp-firstpass.dll*				UnityEngine.CoreModule.dll.mdb*				UnityEngine.PhysicsModule.dll*				UnityEngine.UnityConnectModule.dll.mdb*
Assembly-CSharp.dll*					UnityEngine.CoreModule.xml*				UnityEngine.PhysicsModule.dll.mdb*			UnityEngine.UnityConnectModule.xml*
Ionic.Zip.dll*						UnityEngine.CrashReportingModule.dll*			UnityEngine.PhysicsModule.xml*				UnityEngine.UnityWebRequestAudioModule.dll*
KSPAssets.dll*						UnityEngine.CrashReportingModule.dll.mdb*		UnityEngine.ScreenCaptureModule.dll*			UnityEngine.UnityWebRequestAudioModule.dll.mdb*
KSPTrackIR.dll*						UnityEngine.CrashReportingModule.xml*			UnityEngine.ScreenCaptureModule.dll.mdb*		UnityEngine.UnityWebRequestAudioModule.xml*
MechJeb2.dll						UnityEngine.DirectorModule.dll*				UnityEngine.ScreenCaptureModule.xml*			UnityEngine.UnityWebRequestModule.dll*
MechJeb2.pdb						UnityEngine.DirectorModule.dll.mdb*			UnityEngine.SpriteMaskModule.dll*			UnityEngine.UnityWebRequestModule.dll.mdb*
MechJeb2Tests.dll					UnityEngine.DirectorModule.xml*				UnityEngine.SpriteMaskModule.dll.mdb*			UnityEngine.UnityWebRequestModule.xml*
MechJeb2Tests.pdb					UnityEngine.GameCenterModule.dll*			UnityEngine.SpriteMaskModule.xml*			UnityEngine.UnityWebRequestTextureModule.dll*
Mono.Cecil.dll*						UnityEngine.GameCenterModule.dll.mdb*			UnityEngine.StyleSheetsModule.dll*			UnityEngine.UnityWebRequestTextureModule.dll.mdb*
Mono.Security.dll*					UnityEngine.GameCenterModule.xml*			UnityEngine.StyleSheetsModule.dll.mdb*			UnityEngine.UnityWebRequestTextureModule.xml*
NSubstitute.dll*					UnityEngine.IMGUIModule.dll*				UnityEngine.StyleSheetsModule.xml*			UnityEngine.UnityWebRequestWWWModule.dll*
NSubstitute.xml*					UnityEngine.IMGUIModule.dll.mdb*			UnityEngine.TerrainModule.dll*				UnityEngine.UnityWebRequestWWWModule.dll.mdb*
RedShellSDK.dll*					UnityEngine.IMGUIModule.xml*				UnityEngine.TerrainModule.dll.mdb*			UnityEngine.UnityWebRequestWWWModule.xml*
System.Core.dll*					UnityEngine.ImageConversionModule.dll*			UnityEngine.TerrainModule.xml*				UnityEngine.VRModule.dll*
System.Xml.dll*						UnityEngine.ImageConversionModule.dll.mdb*		UnityEngine.TerrainPhysicsModule.dll*			UnityEngine.VRModule.dll.mdb*
UnityEngine.AIModule.dll*				UnityEngine.ImageConversionModule.xml*			UnityEngine.TerrainPhysicsModule.dll.mdb*		UnityEngine.VRModule.xml*
UnityEngine.AIModule.dll.mdb*				UnityEngine.InputModule.dll*				UnityEngine.TerrainPhysicsModule.xml*			UnityEngine.VehiclesModule.dll*
UnityEngine.AIModule.xml*				UnityEngine.InputModule.dll.mdb*			UnityEngine.TextRenderingModule.dll*			UnityEngine.VehiclesModule.dll.mdb*
UnityEngine.AnimationModule.dll*			UnityEngine.InputModule.xml*				UnityEngine.TextRenderingModule.dll.mdb*		UnityEngine.VehiclesModule.xml*
UnityEngine.AnimationModule.dll.mdb*			UnityEngine.JSONSerializeModule.dll*			UnityEngine.TextRenderingModule.xml*			UnityEngine.VideoModule.dll*
UnityEngine.AnimationModule.xml*			UnityEngine.JSONSerializeModule.dll.mdb*		UnityEngine.UI.dll*					UnityEngine.VideoModule.dll.mdb*
UnityEngine.AudioModule.dll*				UnityEngine.JSONSerializeModule.xml*			UnityEngine.UIElementsModule.dll*			UnityEngine.VideoModule.xml*
UnityEngine.AudioModule.dll.mdb*			UnityEngine.ParticleSystemModule.dll*			UnityEngine.UIElementsModule.dll.mdb*			UnityEngine.WebModule.dll*
UnityEngine.AudioModule.xml*				UnityEngine.ParticleSystemModule.dll.mdb*		UnityEngine.UIElementsModule.xml*			UnityEngine.WebModule.dll.mdb*
UnityEngine.ClothModule.dll*				UnityEngine.ParticleSystemModule.xml*			UnityEngine.UIModule.dll*				UnityEngine.WebModule.xml*
UnityEngine.ClothModule.dll.mdb*			UnityEngine.ParticlesLegacyModule.dll*			UnityEngine.UIModule.dll.mdb*				UnityEngine.dll*
UnityEngine.ClothModule.xml*				UnityEngine.ParticlesLegacyModule.dll.mdb*		UnityEngine.UIModule.xml*				UnityEngine.dll.mdb*
UnityEngine.ClusterInputModule.dll*			UnityEngine.ParticlesLegacyModule.xml*			UnityEngine.UNETModule.dll*				UnityEngine.xml*
UnityEngine.ClusterInputModule.dll.mdb*			UnityEngine.PerformanceReportingModule.dll*		UnityEngine.UNETModule.dll.mdb*				xunit.abstractions.dll*
UnityEngine.ClusterInputModule.xml*			UnityEngine.PerformanceReportingModule.dll.mdb*		UnityEngine.UNETModule.xml*				xunit.dll*
UnityEngine.ClusterRendererModule.dll*			UnityEngine.PerformanceReportingModule.xml*		UnityEngine.UnityAnalyticsModule.dll*			xunit.runner.reporters.net452.dll*
UnityEngine.ClusterRendererModule.dll.mdb*		UnityEngine.Physics2DModule.dll*			UnityEngine.UnityAnalyticsModule.dll.mdb*		xunit.runner.utility.net452.dll*
UnityEngine.ClusterRendererModule.xml*			UnityEngine.Physics2DModule.dll.mdb*			UnityEngine.UnityAnalyticsModule.xml*			xunit.runner.visualstudio.testadapter.dll*
UnityEngine.CoreModule.dll*				UnityEngine.Physics2DModule.xml*			UnityEngine.UnityConnectModule.dll*			xunit.xml*`
```

all that gets ignored by .gitignore though and if you point it at a more reasonable subsection of DLLs it won't need all these...   there might be some way to trim this down to only the necessary ones, but so far it doesn't actually seem to hurt anything (and works on Mac even though none of these are coming from the Launcher.app directory oddly enough...)